### ISSUE TITLE
About  the cosine similarity between the pixel feature and the anchor

### ISSUE THREAD

**Issue by PilgrimMay** [TFboys-lzz/MPSCL#4]
In the paper, logits are defined as the cosine similarity between the pixel feature and the anchor. However,  it seems the code uses "cosine = torch.matmul(anchor_feature, class_center_feas)" directly instead of cosine similarity. Maybe I do not fully understand the paper or code. Could you explain this issue? Thanks a lot.

**Comment by TFboys-lzz** [comment]
Hi, PilgrimMay：
The feature has been normalized in 
https://github.com/TFboys-lzz/MPSCL/blob/bb58bde64e904f167cad9ed7b6f4f64d3977be1e/utils/func.py#L81

**Comment by PilgrimMay** [comment]
Got it! Thanks a lot!
